x/campaign/keeper/msg_server_edit_campaign.go

```diff
@@ -31,8 +31,14 @@ func (k msgServer) UpdateCampaignName(goCtx context.Context, msg *types.MsgUpdat
 			campaign.CoordinatorID,
 		))
 	}
+
 	campaign.CampaignName = msg.Name
```

Comment:
IMO we should also have the option to not update the name
```suggestion
if len(msg.Name) > 0 {
	campaign.CampaignName = msg.Name
}
```